src/main/groovy/grails/mongeez/MongeezGrailsPlugin.groovy

```diff
@@ -48,9 +49,14 @@ class MongeezGrailsPlugin extends Plugin {
         log.debug "Configuring Grails Mongeez Plugin..."
 
         String mongoBeanName = 'mongo'
-        String databaseName = grailsApplication.config.grails.mongodb.databaseName
-        String username = grailsApplication.config.grails.mongodb.username
-        String password = grailsApplication.config.grails.mongodb.password
+        Map<String, Object> mongodbConfigurations = grailsApplication.config.grails.mongeezCreds ?:
+                grailsApplication.config.grails.mongodb
```

Comment:
We should add in README that if `mongeez` configuration is found then it ignores te `mongodb` configuration.

Reply:
Sure.